beets/autotag/mb.py

```diff
@@ -285,6 +287,24 @@ def album_info(release):
     artist_name, artist_sort_name, artist_credit_name = \
         _flatten_artist_credit(release['artist-credit'])
 
+    ntracks = 0
+    for medium in release['medium-list']:
+        ntracks += len(medium['track-list'])
+
+    # for albums with more than 500 tracks
+    if ntracks > 500:
+        recording_list = []
+        for i in range((ntracks // 100) + 1):
```

Comment:
Could also use `range(0, ntracks, RECORDING_CHUNK_SIZE)` to move in "chunks" of 100?

Reply:
Done